### DISCUSSION TITLE
Configure Table is not working

### DISCUSSION THREAD

**Discussion by nikor30** [netbox-community/netbox#6370] [Q&A]
### NetBox version

v.2.11.2

### Python version

3.8

### Steps to Reproduce

    Click on a Section
    Click on Configure
    Change table view or reset to default

### Expected Behavior

It should work :-)

### Observed Behavior

![grafik](https://user-images.githubusercontent.com/4319444/116057699-57db9d80-a67f-11eb-861a-7bf9ae37989c.png)

**Comment by jeremystretch** [comment]
I'm afraid you'll need to provide more detail than that. It works fine for me in v2.11.2 across multiple browsers. That error dialog indicates that the REST API request is failing; see if you can figure out why.

**Comment by nikor30** [comment]
Hmm, it´s strange I ahve also and dev machine cloned from the productive and it doesnt show that problem...

Where can I switch on the debuging to provide more infos ?

Regards Niko

**Comment by jeremystretch** [comment]
You can open your web browser's developer console (usually by pressing F12) to inspect the HTTP requests and Javascript console.

**Comment by nikor30** [comment]
jquery-3.5.1.min.js:2 PATCH https://sdegilvnetbx1.global.web-int.net/api/users/config/ net::ERR_EMPTY_RESPONSE
send @ jquery-3.5.1.min.js:2
ajax @ jquery-3.5.1.min.js:2
(anonymous) @ tableconfig.js:48
dispatch @ jquery-3.5.1.min.js:2
v.handle @ jquery-3.5.1.min.js:2

![grafik](https://user-images.githubusercontent.com/4319444/117107132-5eee6400-ad81-11eb-9ae6-b48ca82e2d41.png)

![grafik](https://user-images.githubusercontent.com/4319444/117107379-c5738200-ad81-11eb-9945-93e1084b63dc.png)


Hope this helps

**Comment by nikor30** [comment]
Hi, now you moved it to discussion, so hopefully here some one can help me on that topic :-)

**Comment by yaigor5** [comment]
Hi, I see this also.
![изображение](https://user-images.githubusercontent.com/48884281/118833709-f4314280-b8c1-11eb-9a74-df33767653a9.png)
After click on "reset" do not add back anything.


![изображение](https://user-images.githubusercontent.com/48884281/118834805-c1d41500-b8c2-11eb-91fe-303d4b4d7be9.png)

**Reply by candlerb** [reply]
@yaigor5: which Netbox version and which browser are you using?

There was a fix in v2.11.2 for this with Chrome. #6248

**Reply by yaigor5** [reply]
v2.11.3 - 2021-05-07 Netbox
Firefox latest version

**Reply by yaigor5** [reply]
In IE (ver11) is the same situation.

**Reply by yaigor5** [reply]
netbox (v2.11.4) - do not any changes

**Reply by candlerb** [reply]
Table column selection works fine for me (Chrome and Firefox, macOS, Netbox v2.11.4).

I might be something wrong with your front-end proxy config (Apache or Nginx).  Can you post your full proxy config?  Are you enforcing any authentication in the front-end proxy?

You showed the "Headers" tab from the developer console, but not the "Response" tab.  The answer might be there.

Also: check carefully the *first* request to /api/users/config/

**Comment by yaigor5** [comment]
Where in DB I can changed it manually?

**Comment by rodvand** [comment]
Try a hard refresh (Ctrl + F5 or similar).

**Reply by yaigor5** [reply]
☺️ I probe in other (incognito) window in firefox. Thanx, I know about refresh and cache clearing.
